yt_dlp/extractor/cda.py

```diff
@@ -351,3 +351,63 @@ def extract_format(page, version):
             extract_format(webpage, resolution)
 
         return merge_dicts(info_dict, info)
+
+
+class CDAFolderIE(InfoExtractor):
+    _MAX_PAGE_NUMBER = 200
+    _VALID_URL = r'https?://(?:www\.)?cda\.pl/[0-9a-zA-Z_]+/folder/(?P<id>[0-9a-z]+)(/vfilm)?'
```

Comment:
Do you have any example URLs where the ID contains letters?
If no, we can use:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?cda\.pl/\w+/folder/(?P<id>\d+)'
```